C#/cs01/tests/13.cs

```diff
@@ -0,0 +1,59 @@
+using NUnit.Framework;
+using Brainchild.Curriculum.Solutions;
+using System;
+namespace Brainchild.Curriculum
+{
+    [TestFixture]
+    public class Test13
+    {
+        [SetUp]
+        public void SetUp()
+        {
+        }
+
+        [Test]
+        public void CheckingGenericClassStringStack()
+        {
+            Stack<string> stringstack = new Stack<string>();
+            stringstack.Push("abc");
+            stringstack.Push("abcd");
+            stringstack.Push("abcde");
+            stringstack.Pop();
+            string[] actual = stringstack.Output();
+            string[] expecting = new string[1000];
+            expecting[0] = "abc";
+            expecting[1] = "abcd";
+            Assert.AreEqual(actual, expecting);
+        }
+        [Test]
+        public void CheckingGenericClassIntStack()
+        {
+            Stack<int> intstack = new Stack<int>();
+            intstack.Push(5);
+            intstack.Push(7);
+            intstack.Push(88);
+            intstack.Pop();
```

Comment:
you should add a test to verify the popped value ? Pop should return the top of the stack element

Reply:
Done.Added a new test for pop up value.